language/en-GB/en-GB.site_menuitems.ini

```diff
@@ -11,3 +15,29 @@ MENU_DOWNLOADS_ITEM5="Language Packs"
 MENU_DOWNLOADS_ITEM6="Technical Requirements"
 MENU_DOWNLOADS_ITEM7="FAQs"
 MENU_DOWNLOADS_ITEM8="API Documentation"
+
+; Menus: Page Titles, Headings and Meta Descriptions
+MENU_DOWNLOADS_HOME_PAGETITLE="Build your website with the CMS Joomla!"
+MENU_DOWNLOADS_HOME_METADESC="Download the free open source CMS Joomla! and start building your awesome website."
+MENU_DOWNLOADS_LATEST_PAGETITLE="Joomla! Downloads - Download the latest version of the CMS"
```

Comment:
`MENU_DOWNLOADS_LATEST_PAGETITLE="Joomla! Downloads - Download the latest version!"`